provisioning/device/samples/solutions/BestPracticeSampleX509/readme.md

```diff
@@ -0,0 +1,128 @@
+# Best Practice Solution Sample with X.509 Authentication
+
+## Objective
+
+This doc demonstrates a real-world example to help you get started with building your own custom IoT cloud solution with X.509 certificate-based authentication. See [here](https://learn.microsoft.com/en-us/azure/iot-dps/concepts-service#Authentication-mechanism) to review available attestation mechanisms.
+
+## Example
+
+Company-X is a manufacturing company that wants to manage various types of products remotely.
```

Comment:
"Contoso" is the usual go-to fictional company Microsoft uses in cases like this, FYI

Reply:
Sure!